pynicotine/gtkgui/userbrowse.py

```diff
@@ -1044,25 +1047,17 @@ def on_file_properties(self, _action, _state, all_files=False):
             self.userbrowses.file_properties.show()
 
     def on_copy_file_path(self, *_args):
-
-        if self.selected_folder is None or not self.selected_files:
-            return
-
-        text = "\\".join([self.selected_folder, next(iter(self.selected_files))])
-        copy_text(text)
+        file_path = self.get_selected_file_path()
+        copy_text(f"{self.user}\\{file_path}")
```

Comment:
Changes to paths should not be included in this PR

Reply:
reverted.